canonical-web-and-design/maas-ui
# Filter select when taking action

## Done
- Filter selected machines when opening a take action form.

## QA
- Select a machine that can be commissioned.
- Using the take action menu open the form.
- The machine list should now be filtered by in:selected.
- Submit the form.
- The machine should transition to commissioning but the update-your-selection warning should not appear (like https://user-images.githubusercontent.com/17980375/80403010-c0e6d480-88b6-11ea-813b-8917aa9eb479.png).
- Click the machine, group or all machines checkbox, the machine list should no longer be filtered

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/1042.
Fixes: https://github.com/canonical-web-and-design/MAAS-squad/issues/1897.
